docs/low-code-spark/spark-streaming/structured-streaming.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Streaming Applications
+id: streaming-apps
+description: Low-code Spark Streaming
+sidebar_position: 5
+tags:
+  - spark
+  - streaming
+---
+
+Prophecy 2.7 introduces native support for streaming data running on Spark Structured Streaming. The streaming capability is available for Python projects. Support for Scala will be added in the future.
+
+Streaming pipelines work differently from batch pipelines:
+
+1. Streaming applications are always running, continuously processing incoming data.
+2. Data is processed in microbatches. (with the exception of Continuous Triggers, which are still an experimental feature in Spark 3.3). More on Continuous Trigger [here](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#continuous-processing)
+3. Streaming applications handle transient data, rather than maintain the entire data. Hence, Real-world aggregations and joins require watermarking for maintaining a limited state.
+4. All Streaming Datasets can be made to behave as Batch datasets using the Spark ForEachBatch. More on ForEachBatch [here](https://spark.apache.org/docs/3.1.1/api/python/reference/api/pyspark.sql.streaming.DataStreamWriter.foreachBatch.html) Note that forEachBatch is ont supported yet.
+   This documentation assumes you are already familiar with how Structured Streaming works. For more information, you can consult the Structured Streaming documentation [here](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html).
+
+## Spark Streaming using Prophecy IDE
+
+![How to Create Streaming Pipeline](./img/create-streaming-pipeline.png)
+Within a Python Project, a User can create a Streaming Pipeline much like creating a Batch Pipeline.
+
+### Creating a Streaming Pipeline
+
+To create a Streaming Pipeline, users can follow a process similar to creating a Batch Pipeline within a Python project. For more on Pipeline creation and understanding Prophecy pipelines, please check [this](/concepts/pipeline) link. However, there are some differences in execution:
+
+1. Partial runs are not possible for streaming applications. Partial run is only allowed on a StreamingTarget Gem.
```

Comment:
If we have documentation explaining what "Partial runs" are, we should link to it here.